platform/ATCmdParser.cpp

```diff
@@ -380,3 +380,44 @@ void ATCmdParser::abort()
 {
     _aborted = true;
 }
+
+bool ATCmdParser::process_oob()
+{
+    if (!_fh->readable()) {
+        return false;
+    }
+
+    int i = 0;
+    while (true) {
+        // Receive next character
+        int c = getc();
+        if (c < 0) {
+            return false;
+        }
+        _buffer[i++] = c;
+        _buffer[i] = 0;
+
+        // Check for oob data
+        struct oob *oob = _oobs;
+        while ( oob ) {
```

Comment:
formatting

Reply:
I'm not seeing a formatting issue on this specific line, may be I'm missing the point, can you please give me details?

Reply:
should be `while (oob) {`, no spaces around conditional arguments